### DISCUSSION TITLE
Question: Can this run automatically without user interaction?

### DISCUSSION THREAD

**Discussion by MrGeorgeK55** [PartialVolume/shredos.x86_64#138] [Q&A] [answered]
Same as title, they are a setting to enable automatic deletion on every hard drive on the computer without user interaction?

**Comment by nuneze54** [comment, marked as answer]
As it says in the readme under the section "How to change the default nwipe options so the change persists between reboots":

> For reference and as of nwipe v0.32.014, listed below are all the options that you can use with nwipe and can place on the kernel command line in grub.cfg as described in the examples above.

      --autonuke          If no devices have been specified on the command line,
                          starts wiping all devices immediately. If devices have
                          been specified, starts wiping only those specified
                          devices immediately.
